RealmTasks Apple/RealmTasks iOS/ViewController.swift

```diff
@@ -135,15 +157,14 @@ final class ViewController<Item: Object, Parent: Object where Item: CellPresenta
                 return
             }
         } else {
-            var row: Int = 0
-            try! items.realm?.write {
-                row = items.filter("completed = false").count
-                items.insert(Item(), atIndex: row)
-            }
+            items.realm?.beginWrite()
```

Comment:
No, `uiWriteNoUpdateList()` takes a block, whereas this _starts_ a transaction that will be open beyond the scope of this function, so a block can't be used.